tuxemon/resources/db/locale/eo.json

```diff
@@ -5,18 +5,18 @@
     "combat_used_x_on_y": "${{name}} estis uzata al ${{target}} de ${{user}}!",
     "combat_call_tuxemon": "Iru, ${{name}}!",
     "combat_opponent_call_tuxemon": "${{user}} sendas ${{name}}!",
-    "combat_state_poison_damage": "${{name}} estis venenigita!",
-    "combat_status_damage": "${{name}} estis damaĝita de ${{status}}!",
+    "combat_state_poison_damage": "${{name}} veneniĝis!",
+    "combat_status_damage": "${{name}} damaĝiĝis pro ${{status}}!",
     "combat_victory": "Vi gajnis!",
     "combat_defeat": "Vi estis venkita!",
-    "combat_capturing success": "Vi kaptis ${{name}}!",
+    "combat_capturing success": "${{name}} estias kaptita!",
     "combat_capturing_fail": "${{name}} liberiĝis!",
     "combat_replacement": "Elektu anstataŭanton!",
```

Comment:
I kept the [older](https://github.com/Clonewayx/Tuxemon/blob/3a85ae69b7507d413ce97eaffaa14b6aa498d0e5/tuxemon/resources/db/locale/eo_EU.json) translation. I remove ```-anto``` as it was a bit weird to me. Now I know why ;)
Moreover [the dictionary Italian-Esperanto](http://www.swahili.it/glossword/index.php?a=term&d=2&q=sostituzione) doesn't use "anstataŭanto"